packages/web/docs/src/pages/product-updates/2023-11-10-laboratory-persist-state.mdx

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Not sure if that isn't too minor to include in the updates

Reply:
I think it can be added, especially if this changes the default behavior and might effect things like security.
@beerose maybe we should mention that this new behavior can be disabled from the editor's settings?